data/config/MLFeatures.json

```diff
@@ -1,9 +1,10 @@
-{ MarkLogicFeatures: [
-            "Entity Services",
-	    "Data Integration",
-            "Data Movement",
-	    "Template Driven Extraction",
-            "SQL"
-	    ]
+{
+  "MarkLogicFeatures": [
```

Comment:
Still looks like it zig zags a little here, not sure why because it looks fine in sublime...